src/Umbraco.ModelsBuilder/Umbraco/LiveModelsProvider.cs

```diff
@@ -133,9 +127,7 @@ public void Dispose()
 
         public static void Install()
         {
-            if (!LiveModelsProvider.IsEnabled)
-                return;
-
+            // always - don't read config in PreApplicationStartMethod
             HttpApplication.RegisterModule(typeof(LiveModelsProviderModule));
```

Comment:
This is what we'll most likely have to alter a bit to detect the original MB is active.